environment/learnrails/run.r

```diff
@@ -0,0 +1,4 @@
+calculator = Calculator.new([2,4,5,1])
```

Comment:
is this working 
filename is  environment/learnrails/run.r

it should be environment/learnrails/run.rb